layouts/shortcodes/cve-feed.html

```diff
@@ -1,9 +1,19 @@
 {{ $feed := getJSON .Site.Params.cveFeedBucket }}
+{{ if eq $feed nil }}
+  {{- if eq hugo.Environment "production" }}
+    {{ errorf "Unable to load CVE feed. Aborting build." }}
+  {{- else -}}
+    {{ warnf "Cannot load CVE feed." }}
+  {{- end -}}
+{{ end }}
 {{ if ne $feed.version "https://jsonfeed.org/version/1.1" }}
   {{ warnf "CVE feed shortcode. KEP-3203: CVE feed does not comply with JSON feed v1.1." }}
 {{ end }}
 <table class="security-cves">
+  <!-- Handle case if CVE feed is not available. For example for offline build -->
+  {{ if ne $feed nil }}
```

Comment:
_Optional improvement:_ Instead of just displaying an empty table with headers, we could enhance the user experience by showing a helpful message, perhaps in a _note_, indicating that the offline preview does not support the CVE Feed.

Reply:
There is another way.

If we change the fetch to use https://gohugo.io/functions/resources/getremote/, then we can _attempt_ to fetch offline and warn at build time. And, yes, we can substitute something in.

For a production build, we should error out instead of warning.

Reply:
User will receive a warning message in build log. I guess that build should works offline if some of resources are unavailable. 

Yes we won't display some content that require remote files but other content will be ok and editable